frontend/www/js/omegaup/components/problem/CollectionProblem.vue

```diff
@@ -2,23 +2,21 @@
   <div class="col-sm-4 mt-2">
     <div class="border border-dark">
       <div class="row">
-        <div class="col-sm-4 text-center pt-2">
+        <div class="col-sm-4 d-flex align-items-center justify-content-end">
           <h1>
-            <font-awesome-icon
-              :icon="['fas', getProblemLevelIcon(levelTagAlias)]"
-            />
+            <slot name="icon"></slot>
           </h1>
         </div>
-        <div class="col-sm-8 text-center" style="height: 112px">
+        <div class="col-sm-8 d-flex align-items-center" style="height: 112px">
           <p>
-            <strong>{{ name }}</strong> <br />
-            {{ problemCount }} {{ T.wordsProblems }}
+            <strong>{{ title }}</strong> <br />
+            {{ problemCount }} <slot name="problems"></slot>
```

Comment:
```suggestion
            <slot name="problem-count"></slot>
```

creo que es mejor que el otro componente provea la cadena completa. si no, se va a mostrar como `0` en las listas estáticas. eso significa que en vez de la palabra "problemas" ahora hay que hacer una cadena de traducción que sea

```
problemCollectionProblemCount = "%(count) problems"
```

y usar `ui.formatString(T.problemCollectionProblemCount, {count: collection.problems_per_tag})` para desplegarlo.